docs/HowTos/GitAndGitHub/git_setup.md

```diff
@@ -0,0 +1,17 @@
+# Git setup
+
+Make sure you set up git on every machine (including servers such as HPCs for example) you work on.
```

Comment:
I think it would be good to have an introductory sentence here explaining the purpose of this section. Something like:

"git has many configuration options, this section covers some of the basic options that are recommended"

Also I would change the sentence to this:

"Make sure you set these configuration options on every machine you routinely use, including remote servers such as HPCs."

Reply:
Done